R/init.R

```diff
@@ -15,10 +15,11 @@
 #' an end-user, don't use this function, but instead this module.
 #'
 #' @param data (`TealData` or `TealDataset` or `TealDatasetConnector` or `list` or `data.frame`
-#' or `MultiAssayExperiment`, `teal_data`)\cr
+#' or `MultiAssayExperiment`, `teal_data`, `teal_data_module`)\cr
```

Comment:
```suggestion
#' or `MultiAssayExperiment` or `teal_data` or `teal_data_module`)\cr
```

Reply:
Let's skip it. We will fix this param description anyway soon